src/components/NavBar/index.tsx

```diff
@@ -48,7 +49,9 @@ const NavBar: React.FC = () => {
       </FlexContainer>
       <ButtonContainer>
         <StyledButton buttonText="Filters">SomeContent</StyledButton>
-        <StyledButton buttonText="Details">SomeContent</StyledButton>
+        <StyledButton buttonText="Details" data-cy="details-button">
```

Comment:
It'd be awesome to have a badge/circle/icon of some sort with the number of filters applied when the dropdown is closed. Maybe worth a new ticket or design discussion though?